### ISSUE TITLE
Content: Mark up language change

### ISSUE THREAD

**Issue by terotik** [kausaltech/kausal-watch-ui#37]
If there are different languages in content, the language should be marked with lang="sv" tag
(Screenreader should pronounce the language correctly)

WCAG 2.1  3.2.1 AA
